portfolio/src/main/webapp/script.js

```diff
@@ -15,12 +15,12 @@
 /**
  * Check the validity of the parameters.
  */
-function check(element, classAttribute, functionName) {
+function check(element, classAttribute) {
```

Comment:
note that you also have to update the callsites and remove the extra argument.

Reply:
Sure, I completely forgot.